timetagger/pages/login.md

```diff
@@ -3,11 +3,13 @@
 <script src='./app/tools.js'></script>
 
 <script>
-async function login(payload) {
+async function login(payload, trial_run) {
```

Comment:
Maybe rename this to silent, because that's what seems to be the effect :)

Reply:
Renamed, great suggestion :+1:  That's exactly what it is doing.